guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -112,11 +112,10 @@ Use `grep tar_file /etc/foreman-installer/scenarios.d/capsule-answers.yaml` on y
 +
 . The `rubygem-foreman_maintain` is installed from the {Project} Maintenance repository or upgraded from the {Project} Maintenance repository if currently installed.
 +
-Ensure {SmartProxy} has access to `{RepoRHEL7ServerSatelliteMaintenanceProductVersion}` and execute:
-+
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-maintain} self-upgrade
+# subscription-manager repos --enable {RepoRHEL7ServerSatelliteMaintenanceProductVersion}
+# yum --disableplugin=foreman-protector update rubygem-foreman_maintain {foreman-maintain}
```

Comment:
We have a convention to only have one command per code block.

Reply:
Does the second command have a different job? The reason is I have come across multi-line commands per block.

Reply:
They do have different intentions. If you look at the old version there is the line "Ensure {SmartProxy} has access to `{RepoRHEL7ServerSatelliteMaintenanceProductVersion}`". The subscription-manager command does exactly that. The other command updates the foreman-/satellite-maintain application.

So you can keep it in a single command. Then you should keep the old description and change `execture` to `update {foreman-maintain}`.

The alternative is to describe each command separately.

Either way, I'd drop the sentence on line 113.